docs/faq.md

```diff
@@ -151,20 +151,20 @@ The default polling interval is 3 minutes (180 seconds) with a configurable jitt
 You can change the setting by updating the `timeout.reconciliation` value and the `timeout.reconciliation.jitter` in the [argocd-cm](https://github.com/argoproj/argo-cd/blob/2d6ce088acd4fb29271ffb6f6023dbb27594d59b/docs/operator-manual/argocd-cm.yaml#L279-L282) config map. If there are any Git changes, Argo CD will only update applications with the [auto-sync setting](user-guide/auto_sync.md) enabled. If you set it to `0` then Argo CD will stop polling Git repositories automatically and you can only use alternative methods such as [webhooks](operator-manual/webhook.md) and/or manual syncs for deploying applications.
 
 
-## Why Are My Resource Limits `Out Of Sync`?
+## Why is my ArgoCD application `Out Of Sync` and keep syncing when there are no actual changes in resource limits or other fields with unit values?
```

Comment:
```suggestion
## Why does my ArgoCD application keep syncing when there are no actual changes in resource limits or other fields with unit values?
```

how about this?

Reply:
I'd keep about out of sync, to also cover people who have auto sync disabled and still wondering about the issue.

Reply:
Or this one 

```suggestion
## Why is my ArgoCD application `Out Of Sync` when there are no actual changes to the resource limits (or other fields with unit values)?
```

Reply:
Done